### PR TITLE
Include touchup, fixes #41

### LINKED ISSUE #41: what's the deal with _inc/contrib/?
Is that directory used for anything, or is it a red herring? Seems like unnecessary cruft to have to wade through...

### PR DESCRIPTION
**Pull request by mikesherov**
1. get rid of unnecessary contrib directory from GenSHi (addresses #41)
2. begin using autoload when possible
3. make it easier to be in debug mode

### CODE REVIEW

**Comment by janmoesen** [comment]
FYI, I did the same as your commit e26882c (127.0.0.1 === debug) in commit 9ca867a.

**Comment by mathiasbynens** [comment]
I want to pull this in, but I’m getting merge conflicts. HALP!1

File: _inc/start.php [hunk 1]
```diff
@@ -1,11 +1,15 @@
 <?php
 
+function __autoload($classname){
+	include_once(strtolower($classname).'.php');
```
Comment (mathiasbynens) [inline _inc/start.php]: Would you mind adding spaces around the `.` please? Thanks :)

**Comment by mikesherov** [comment]
rebased, and style changes made. should merge cleanly now.

**Comment by mathiasbynens** [comment]
Thanks a lot!

**Comment by mikesherov** [comment]
no problem!

**Pull request merged**